web/packages/teleport/src/Notifications/fixtures.ts

```diff
@@ -0,0 +1,143 @@
+/**
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { NotificationSubKind } from 'teleport/services/notifications';
+import { Notification } from 'teleport/services/notifications';
+
+export const notifications: Notification[] = [
+  {
+    id: '1',
+    title: 'joe approved your access request for 5 resources.',
+    subKind: NotificationSubKind.AccessRequestApproved,
+    createdDate: new Date(Date.now() - 30 * 1000), // 30 seconds ago
```

Comment:
Let's use `date-fns` for date calculations:
```suggestion
    createdDate: subSeconds(Date.now(), 30),
```